ufmt/cli.py

```diff
@@ -21,6 +22,22 @@ def init_logging(*, debug: bool = False) -> None:
     logging.getLogger("blib2to3").setLevel(logging.WARNING)
 
 
+def echo_results(results: List[Result], diff: bool = False) -> bool:
```

Comment:
Consider not giving `diff` a default and being explicit on the one line that uses it that way.